docs/components/bucket.md

```diff
@@ -42,10 +42,11 @@ Flags:
       --version            Show application version.
       --log.level=info     Log filtering level.
       --log.format=logfmt  Log format to use.
-      --gcloudtrace.project=GCLOUDTRACE.PROJECT
+      --tracing.type=noop  Tracer to use (stackdriver/jaeger/noop).
+      --stackdriver.project=GCLOUDTRACE.PROJECT
                            GCP project to send Google Cloud Trace tracings to.
                            If empty, tracing will be disabled.
-      --gcloudtrace.sample-factor=1
+      --stackdriver.sample-factor=1
```

Comment:
Also it would be great to move most of configuration to one file to have prometheus-like configuration approach. Its convenient to have all flags in one place.

Reply:
I think it's a bigger decision. We chose not to do it as: 
* We have many smaller components instead of big single one like Prometheus
* Most of options are not dynamic. Adding this dynamicity is enourmous amount of work
* Objstore is separate to tracing. And it could be multiple of objstore configs as well, so it's hard to unifiy.

Reply:
But someday (:

Reply:
Okay, got it, thanks! :)